i3_workspace_names_daemon.py

```diff
@@ -57,9 +58,9 @@ def rename(i3, e):
         workspaces = i3.get_tree().workspaces()
         # need to use get_workspaces since the i3 con object doesn't have the visible property for some reason
         workdicts = i3.get_workspaces()
-        visible = [workdict['name'] for workdict in workdicts if workdict['visible']]
+        visible = [workdict.name for workdict in workdicts if workdict.visible]
```

Comment:
thanks a  lot @rand0x0m @CastixGitHub. Can either of you comment if  this change also works with `i3ipc` `v1.x`? 

If it is not backwards compatible that's fine but we should pin the dependency  to `i3ipc >=2.x.y`.